config.default.yaml

```diff
@@ -456,10 +456,12 @@ fossil_reserves:
   oil: 100 #TWh Maybe redundant
 
 export:
-  h2export: [10] # Yearly export demand in TWh
+  h2export: [10] # Yearly export demand in TWh. Only considered, if ["export"]["endogenous"] is set to false
+  endogenous: false # If true, the export demand is endogenously determined by the model
+  endogenous_price: 400 # EUR/MWh # Market price, for wich the hydrogen for endogenous exports is sold. Only considered, if ["export"]["endogenous"] is set to true.
   store: true # [True, False] # specifies whether an export store to balance demand is implemented
   store_capital_costs: "no_costs" # ["standard_costs", "no_costs"] # specifies the costs of the export store. "standard_costs" takes CAPEX of "hydrogen storage tank type 1 including compressor"
-  export_profile: "ship" # use "ship" or "constant"
+  export_profile: "ship" # use "ship" or "constant". Currently only possible when ["export"]["endogenous"] is set to false
```

Comment:
Could it probably make sense to group `export_profile` parameter together with other parameters which are being considered with `endogenous: false`, with `h2export`, in particular? My feling is that it would make the fragment a bit easier to comprehend